### DISCUSSION TITLE
Idk if this is app bug or keyboard one.

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [florisboard/florisboard#1923] [Feedback]
Thats y i am asking this here.

Sometimes it haopens randomly in 1 or two apps.

E.g tachiyomi
https://user-images.githubusercontent.com/38470944/176422700-8b9f8ffb-3487-4ce2-b041-95bf4426712c.mp4

If this is bug i can open issue

**Comment by patrickgold** [comment]
What exactly is the issue? I just see a black screen, so I am missing some context to evaluate what's going on.

**Reply by HikaruIchijyo** [reply]
Check now it should show.

Issue is keyboard pops up & then closes it is fast u need to see when i click on back.

Updated the video chk again plz thanks

**Comment by HikaruIchijyo** [comment]
I guess this is bug since it happened on other apps too like blackhole droidify etc

Should i create the ticket?
